Menu Item fix

ClementPadovani opened the pull request:
This fixes the following issue:

https://github.com/supermarin/Alcatraz/issues/340

jurre commented:
Thanks @ClementPadovani :)

@supermarin LGTM?

supermarin commented on Alcatraz/Alcatraz.m:
```diff
@@ -54,13 +54,25 @@ - (id)initWithBundle:(NSBundle *)plugin {
     return self;
 }
 
+- (void) applicationDidFinishLaunching: (NSNotification *) notification
```
Can you please rename this to something like `xcodeDidFinishLaunching`?

supermarin commented:
Thanks for the contribution @ClementPadovani!

Looks good, except for the comment and it seems like tabs are used instead of spaces.
Do you mind converting the indentation to spaces like the rest of the codebase?

ClementPadovani commented:
@supermarin You’re welcome. Updated the code!

ClementPadovani commented:
@dmiedema Thanks!

wczekalski commented:
A similar solution didn't work for me in XVim in 6.4 and had to be replaced with [this](https://github.com/XVimProject/XVim/pull/821).

supermarin commented:
@ClementPadovani sorry for the delay on this.

I looked on it a few times, and it seems like tabs are still used instead of spaces:
<img width="935" alt="screen shot 2015-11-02 at 12 54 40 pm" src="https://cloud.githubusercontent.com/assets/627285/10894031/ede0e08c-8160-11e5-81f7-106a9853b467.png">

If you wouldn't mind correcting them, that'd be awesome and we're ready to merge.

ClementPadovani commented:
@supermarin Hey Marin, weird I thought I had replaced those. I just replaced the tabs with spaces in this commit: https://github.com/ClementPadovani/Alcatraz/commit/5c381242628b1279aca4f85afa07ea5e69625016

supermarin commented:
@ClementPadovani thanks for addressing!

Now just one more time, the upper method :)
<img width="931" alt="screen shot 2015-11-03 at 9 46 14 am" src="https://cloud.githubusercontent.com/assets/627285/10916328/cf36b918-820f-11e5-9af2-f7f96022a2ca.png">

ClementPadovani commented:
@supermarin Sorry for missing the last one, should have fixed.

supermarin commented:
@ClementPadovani thanks for the contribution! :beers:
